<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="DESKTOP-ECUBPED" failures="3" tests="16" name="TimeStreamGroup.Reconstream.Login" time="13.390" errors="0" timestamp="02 Jun 2023 08:18:48 GMT" skipped="0">
  <testcase classname="TimeStreamGroup.Reconstream.Login" name="ValidateThePageTitle" time="0.073"/>
  <testcase classname="TimeStreamGroup.Reconstream.Login" name="ValidateThatSignInHeadingPersistOnTheLoginPage" time="0.400"/>
  <testcase classname="TimeStreamGroup.Reconstream.Login" name="ValidateThatLoginTextPersistOnTheLoginPage" time="0.294"/>
  <testcase classname="TimeStreamGroup.Reconstream.Login" name="ValidateUserNameFieldPersistOnTheLoginPage" time="0.523"/>
  <testcase classname="TimeStreamGroup.Reconstream.Login" name="ValidatePasswordFieldPersistOnTheLoginPage" time="0.087"/>
  <testcase classname="TimeStreamGroup.Reconstream.Login" name="ValidateForGotPasswordPersistOnTheLoginPage" time="0.103"/>
  <testcase classname="TimeStreamGroup.Reconstream.Login" name="ValidateSigInButtonPersistOnTheLoginPage" time="0.109"/>
  <testcase classname="TimeStreamGroup.Reconstream.Login" name="ValidateUserNameFieldIsEnabled" time="0.289"/>
  <testcase classname="TimeStreamGroup.Reconstream.Login" name="ValidatePasswordFieldEnabled" time="0.089"/>
  <testcase classname="TimeStreamGroup.Reconstream.Login" name="ValidateLoginWithEmptyStrtings" time="0.495"/>
  <testcase classname="TimeStreamGroup.Reconstream.Login" name="ValidateLoginWithEmptyUserName" time="1.507"/>
  <testcase classname="TimeStreamGroup.Reconstream.Login" name="ValidateLoginWithEmpPassword" time="0.426"/>
  <testcase classname="TimeStreamGroup.Reconstream.Login" name="ValidateLoginWithWrongCombination" time="1.845">
    <failure message="expected:&lt;Invalid [Username or P]assword&gt; but was:&lt;Invalid [username or p]assword&gt;" type="junit.framework.ComparisonFailure">
      <![CDATA[junit.framework.ComparisonFailure: expected:<Invalid [Username or P]assword> but was:<Invalid [username or p]assword>
at junit.framework.Assert.assertEquals(Assert.java:100)
at junit.framework.Assert.assertEquals(Assert.java:107)
at TimeStreamGroup.Reconstream.Login.ValidateLoginWithWrongCombination(Login.java:222)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:64)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:564)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- ValidateLoginWithWrongCombination -->
  <testcase classname="TimeStreamGroup.Reconstream.Login" name="ValidateLoginWithWrongCombination" time="2.534">
    <failure message="expected:&lt;Invalid [Username or P]assword&gt; but was:&lt;Invalid [username or p]assword&gt;" type="junit.framework.ComparisonFailure">
      <![CDATA[junit.framework.ComparisonFailure: expected:<Invalid [Username or P]assword> but was:<Invalid [username or p]assword>
at junit.framework.Assert.assertEquals(Assert.java:100)
at junit.framework.Assert.assertEquals(Assert.java:107)
at TimeStreamGroup.Reconstream.Login.ValidateLoginWithWrongCombination(Login.java:222)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:64)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:564)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- ValidateLoginWithWrongCombination -->
  <testcase classname="TimeStreamGroup.Reconstream.Login" name="ValidateLoginWithWrongCombination" time="1.628">
    <failure message="expected:&lt;Invalid [Username or P]assword&gt; but was:&lt;Invalid [username or p]assword&gt;" type="junit.framework.ComparisonFailure">
      <![CDATA[junit.framework.ComparisonFailure: expected:<Invalid [Username or P]assword> but was:<Invalid [username or p]assword>
at junit.framework.Assert.assertEquals(Assert.java:100)
at junit.framework.Assert.assertEquals(Assert.java:107)
at TimeStreamGroup.Reconstream.Login.ValidateLoginWithWrongCombination(Login.java:222)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:64)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:564)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- ValidateLoginWithWrongCombination -->
  <testcase classname="TimeStreamGroup.Reconstream.Login" name="ValidateLoginWithCorrectUserNameAndPassword" time="2.988"/>
</testsuite> <!-- TimeStreamGroup.Reconstream.Login -->
